src/browser/atom-application.coffee

```diff
@@ -517,6 +517,8 @@ class AtomApplication
     return {pathToOpen} unless pathToOpen
     return {pathToOpen} if fs.existsSync(pathToOpen)
 
+    pathToOpen = pathToOpen.replace(/[:\s]*$/, '')
```

Comment:
I _think_ this isn't too aggressive and won't cause problems, but not 100% sure (because regexes make me :cold_sweat:).